tensorflow_addons/losses/BUILD

```diff
@@ -28,3 +28,30 @@ py_test(
     ],
     srcs_version = "PY2AND3",
 )
+
+py_library(
+    name = "lifted_py",
+    srcs = [
+        "__init__.py",
+        "python/__init__.py",
+        "python/lifted.py",
+        "python/metric_learning.py",
+    ],
+    srcs_version = "PY2AND3",
+    deps = [
+        "//tensorflow_addons/utils:utils_py",
+    ],
+)
+
+py_test(
+    name = "lifted_py_test",
+    size = "small",
+    srcs = [
+        "python/lifted_test.py",
+    ],
+    main = "python/lifted_test.py",
+    deps = [
+        ":losses_py",
+    ],
```

Comment:
The build config seems not totally correct. Please use bazel to run tests at first and make sure all pass, see more details at: 
+ https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md
+ https://docs.bazel.build/versions/master/bazel-overview.html

Reply:
You were right i was building the test incorrectly, thanks @facaiy